providers-sdk/v1/lr/go.go

```diff
@@ -167,7 +168,11 @@ func NewResource(runtime *plugin.Runtime, name string, args map[string]*llx.RawD
 		}
 
 		if res != nil {
-			id := name+"\x00"+res.MqlID()
+		  mqlId := res.MqlID()
+			if mqlId == "" {
+			  log.Warn().Msgf("resource %s has no MQL ID defined, this is usually a dev issue with the resource, please open a GitHub issue at https://github.com/mondoohq/cnquery/issues", name)
```

Comment:
nit: Perhaps remove this part from the warninig:
```suggestion
			  log.Warn().Msgf("resource %s has no MQL ID defined, this is usually an issue with the resource, please open a GitHub issue at https://github.com/mondoohq/cnquery/issues", name)
```

Reply:
done